api/v1.0/swipegames-integration/api.yaml

```diff
@@ -0,0 +1,400 @@
+      description: |
+        This endpoint is used to place a refund for bet or win action.
+        We send refund to the client side in case of Bet 500 error or timeout, also
+        in case when player cancels the game.
+
+        After timeout or any error this request will be retrying until we receive 200(OK) response.
+      parameters:
+        - name: X-REQUEST-SIGN
+          in: header
+          required: true
+          description: Request signature (read more in the Authentication section)
+          schema:
+            type: string
+
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/RefundRequest"
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/RefundResponse"
+        400:
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: "../common-components.yaml#/components/schemas/ErrorResponseWithAction"
+        401:
+          description: Wrong signature
+          content:
+            application/json:
+              schema:
+                $ref: "../common-components.yaml#/components/schemas/ErrorResponseWithAction"
+        404:
+          description: User with this game session ID not found
+          content:
+            application/json:
+              schema:
+                $ref: "../common-components.yaml#/components/schemas/ErrorResponseWithAction"
+        500:
+          description: Server error
+          content:
+            application/json:
+              schema:
+                $ref: "../common-components.yaml#/components/schemas/ErrorResponseWithAction"
+
+components:
+  schemas:
+    BalanceResponse:
+      type: object
+      required:
+        - balance
+      properties:
+        balance:
+          description: |
+            The player's balance in selected currency **main** units (**not cents**).
+            Currency selected by the client during the `Create New Game` call.
+            We support 2 decimal places for all fiat currencies.
+          type: string
+          example: "100.10"
+    BetRequest:
+      type: object
+      required:
+        - type
+        - sessionID
+        - amount
+        - txID
+      properties:
+        type:
+          description: |
+            The type of the bet.
+            - `regular` type means regular bet,
+            - `free` type means free bet (see Free Rounds section).
+          type: string
+          enum:
+            - regular
+            - free
+          example: regular
+        sessionID:
+          type: string
+          description: |
+            The Game Session's ID (external). Provided by client via `Create New Game` call.
+          example: some_external_session_id
+        amount:
+          description: |
```

Comment:
Make sure to set amount validation (positive values, max limits)

Reply:
Also not sure about currency and its format validation rules, potentially next step?